.azure-pipelines/build-template.yml

```diff
@@ -110,7 +110,7 @@ jobs:
         make USERNAME=admin $CACHE_OPTIONS SONIC_BUILD_JOBS=$(nproc) target/sonic-${{ parameters.platform }}.bin
       fi
     displayName: 'Build sonic image'
-  - template: cleanup.yml
   - publish: $(System.DefaultWorkingDirectory)/
     artifact: sonic-buildimage.${{ parameters.platform }}
     displayName: "Archive sonic image"
+  - template: cleanup.yml
```

Comment:
i do not understand why we move this.

Reply:
@lguohan , because the cleanup script is changed to remove the entire working folder including the target folder, if not move the last step, no targets will be uploaded.